randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -269,23 +269,17 @@ Extra - room_id: [9]
           Morph Ball
           Any of the following:
               All of the following:
-                  Space Jump
+                  # Sane method
                   Any of the following:
-                      Power Bombs
-                      All of the following:
-                          Morph Ball Bomb Data
-                          Any of the following:
-                              Jump Bombjump (Expert)
-                              Screw Attack and Knowledge (Beginner)
-              All of the following:
-                  Hi-Jump
+                      # Climbing Requirements
+                      Space Jump
+                      Hi-Jump and Wall Jump (Intermediate)
                   Any of the following:
-                      Power Bombs
-                      All of the following:
-                          Morph Ball Bomb Data
-                          Any of the following:
-                              Jump Bombjump (Advanced)
-                              Screw Attack and Knowledge (Beginner) and Wall Jump (Intermediate)
+                      # Bomb Block Requirements
+                      Power Bomb Data
+                      Screw Attack and Knowledge (Beginner)
+              # Cursed method https://youtu.be/Cqf5mZg1NZc
+              Hi-Jump and Morph Ball Bomb Data and Jump Bombjump (Hypermode) and Mid-Air Morph (Intermediate) and Wall Jump (Intermediate)
```

Comment:
🔧 Common `(Hi-Jump + Wall Jump) OR Space Jump` (the space jump part is missing on the cursed logic here) can be pulled out to Morph Ball's level

Reply:
afaik the cursed method hard requires hi jump to get the jump bomb jumps to break the blocks. 
you can't get it NHJ with just space. so its not included here (also why HJ+walljump) isn't grouped with the morph ball

Reply:
unless you mean pulling specifically (walljump OR space jump). I think that could work.

Reply:
nvm I missed that I could jump keep hi jump in the jbj method. fixed